rsocket/ResumeManager.h

```diff
@@ -2,7 +2,73 @@
 
 #pragma once
 
+#include <folly/Optional.h>
+
+#include "rsocket/framing/Frame.h"
+#include "rsocket/framing/FrameTransport.h"
+
+namespace folly {
+class IOBuf;
+}
+
 namespace rsocket {
 
-class ResumeManager {};
+// Applications desiring to have cold-resumption should implement a
+// ResumeManager interface.  By default, an in-memory implementation
+// of this interface (InMemResumeManager) will be used by RSocket.
+class ResumeManager {
+ public:
+  virtual ~ResumeManager(){};
+
+  // The following methods will be called for each frame which is being
+  // sent/received on the wire.  The application should implement a way to
+  // store the sent and received frames in persistent storage.
+  virtual void trackReceivedFrame(
+      const folly::IOBuf& serializedFrame,
+      const FrameType frameType,
```

Comment:
Drop the `const` for `FrameType`, `StreamId`, and `folly::Optional<>`.  They're value types and aren't affected by it.